samples/create_scheduled_query.py

```diff
@@ -68,11 +68,15 @@ def sample_create_transfer_config(project_id, dataset_id, authorization_code="")
             },
             "schedule": "every 24 hours",
         },
-        bigquery_datatransfer_v1.types.TransferConfig(),
+        datatransfer_v1.types.TransferConfig.pb()(),
```

Comment:
Is there a more clean version of this that doesn't require the user to construct a raw protobuf message?

`google.protobuf.json_format.ParseDict()` currently fails if `TransferCOnfig()` instance is passed in directly (AttributeError: 'DESCRIPTOR').

Reply:
I think I used ParseDict because the former client couldn't take a dictionary without failing. Maybe it's fixed now and you don't have to manually call ParseDict?

Reply:
Aha, it seems that one can construct a `TransferConfig` instance directly, the test passes:

```py
    transfer_config = datatransfer_v1.types.TransferConfig(
        **{
            "destination_dataset_id": dataset_id,
            "display_name": "Your Scheduled Query Name",
            "data_source_id": "scheduled_query",
            "params": {
                "query": query_string,
                "destination_table_name_template": "your_table_{run_date}",
                "write_disposition": "WRITE_TRUNCATE",
                "partitioning_field": "",
            },
            "schedule": "every 24 hours",
        }
    )
```

(it's even cleaner to provide data as kwargs instead of unpacking a dict, that was just a quick test that I made)

Reply:
^ I'd also recommend the kwargs approach, you get hints and such in IDEs which is nicer than a big dictionary. 😄